src/simplewallet/simplewallet.cpp

```diff
@@ -3600,14 +3614,31 @@ bool simple_wallet::print_ring_members(const std::vector<tools::wallet2::pending
   return true;
 }
 //----------------------------------------------------------------------------------------------------
-bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::string> &args_)
+bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::string> &args_, size_t try_lowest_priority)
```

Comment:
This gets passed a bool, so I guess this assumes 1 for bool ? That's a bit ugh.

Reply:
It's compliant with The C++ Standard:

> 7.8 Integral conversions [conv.integral]
> 
> 4 If the destination type is `bool`, see 7.14. If the source type is bool, the value `false` is converted to zero and the value `true` is converted to one.

http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2017/n4659.pdf

Of course I can rewrite it using `? 1 : 0`, if desired.

Reply:
Strict adherence to the standard doesn't mean it automatically gets readable and straightforward :)

Reply:
OK, changed as suggested.